packages/client/src/rpc/modules/debug.ts

```diff
@@ -457,4 +460,36 @@ export class Debug {
     this.client.config.logger.configure({ level: logLevels[level] })
     return `level: ${this.client.config.logger.level}`
   }
+
+  /**
+   * Sets the current head of the local chain by block number. Note, this is a
+   * destructive action and may severely damage your chain. Use with extreme
+   * caution.
+   * @param blockOpt Block number or tag to set as head of chain
+   */
+  async setHead(params: [string]) {
+    const [blockOpt] = params
+    if (blockOpt === 'pending') {
+      throw {
+        code: INVALID_PARAMS,
+        message: `"pending" is not supported`,
+      }
+    }
+
+    let headHash = await this.service.skeleton?.headHash()
+    const oldHead = headHash ? bytesToHex(headHash!) : undefined
+    const block = await getBlockByOption(blockOpt, this.chain)
+    try {
+      await this.service.skeleton?.setHead(block, true)
+      await this.service.execution.setHead([block])
+    } catch (e) {
+      throw {
+        code: INTERNAL_ERROR,
+      }
+    }
+    headHash = await this.service.skeleton?.headHash()
+    const newHead = headHash ? bytesToHex(headHash!) : undefined
+
+    return `oldHead: ${oldHead} - newHead: ${newHead}`
```

Comment:
One thought I had here is it would be worth making our response value match Geth's as well.  Strings like above are pretty non-standard for RPC responses and you can't parse them programmatically without having to do fancy string parsing.

Reply:
Yea, I think you're right will remove. It was a convenient way to develop, but will change it to match.